modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/enricher/SampleDataDeclarationEnricherTestCase.java

```diff
@@ -135,7 +165,21 @@ private void assertWithRequiredParameter(OperationDeclaration operationDeclarati
     }
   }
 
+  private void assertAliasedParameter(OperationDeclaration operationDeclaration, String alias, String name) {
```

Comment:
Was this test just moved, or has it changed?

Reply:
Just moved,  I usially prefer the auxiliary methods and the end of the file